.github/ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,9 @@
 <!-- *Before creating an issue please make sure you are using the latest version of mongoose -->
 
+<!-- *Having an issue connecting to mongoose? Check here https://mongoosejs.com/docs/faq.html#operation-buffering-timed-out -->
+
+<!-- *Is a function being used against your will? Check here https://mongoosejs.com/docs/faq.html#not-a-function -->
```

Comment:
Change this to "getting a `x.$__y is not a function` error? Read this FAQ entry: ..."